plugin/jsonSchema.go

```diff
@@ -0,0 +1,39 @@
+package plugin
+
+import (
+	"github.com/ekinanp/jsonschema"
+)
+
+// JSONSchema represents a JSON schema
+type JSONSchema = jsonschema.Schema
```

Comment:
The type alias and the schemas here prevent plugin authors from having to learn about the `jsonschema` library. That's why this file's here (and why it is useful).

Note that Alec did get back to me on one of my jsonschema issues (https://github.com/alecthomas/jsonschema/issues/41) so may end up filing an issue to properly fix the `jsonschema` library and merge those fixes upstream once some of our other in-priority work's finished.